src/index.html

```diff
@@ -1,29 +1,88 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="UTF-8">
-  <title>Activity Tracker</title>
-  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap" rel="stylesheet" />
-</head>
-
-<body>
-  <h1 id="headerText">Activity Tracker</h1>
-  <section class="main">
-    <section class="main-sidebar-body">
-      <section class="sidebar-container" id="user-sidebar"></section>
-    </section>
-    <section class="body-main-infoContainter">
-      <section class="main-column-hydration"></section>
-      <section class="main-column-activity"></section>
-      <section class="main-column-sleep">
+  <head>
```

Comment:
I love how much cleaner this is, than original index.html
I think its going to be easier to work with great work